### PR TITLE
Complete Apache instructions

### PR DESCRIPTION
**Pull request by crees**
Do I still need a CHANGELOG entry for a docs fix?

### CODE REVIEW

**Comment by clokep** [comment]
> Do I still need a CHANGELOG entry for a docs fix?

They're preferred, yes. It helps people realize the docs were updated when taking a look at the release notes.

**Comment by crees** [comment]
Changelog entry done.

File: docs/reverse_proxy.md [hunk 1]
```diff
@@ -111,9 +117,13 @@ example.com:8448 {
     ProxyPassReverse /_synapse/client http://127.0.0.1:8008/_synapse/client
 </VirtualHost>
 
+Listen 8448
```
Comment (anoadragon453) [inline docs/reverse_proxy.md:120]: Just for my own curiosity, this is indeed required as per https://httpd.apache.org/docs/2.4/bind.html.

> To implement a VirtualHost, the server must first be told to listen to the address and port to be used. Then a <VirtualHost> section should be created for the specified address and port to set the behavior of this virtual host. Note that if the <VirtualHost> is set for an address and port that the server is not listening to, it cannot be accessed.

File: docs/reverse_proxy.md [hunk 2]
```diff
@@ -98,10 +98,16 @@ example.com:8448 {
 
 ### Apache
 
+Enable `mod_proxy`, `mod_proxy_http`, `mod_proxy_balancer`, `mod_headers`
+and `mod_lbmethod_byrequests`.
+
 ```
 <VirtualHost *:443>
     SSLEngine on
-    ServerName matrix.example.com;
```
Comment (anoadragon453) [inline docs/reverse_proxy.md:104]: Thanks for catching, `;` are indeed not a thing in Apache configs. And indeed https://httpd.apache.org/docs/2.4/mod/core.html#servername mentions to include a port for `ServerName` if possible:

> For optimal reliability and predictability, you should specify an explicit hostname and port using the ServerName directive.

File: docs/reverse_proxy.md [hunk 3]
```diff
@@ -98,10 +98,16 @@ example.com:8448 {
 
 ### Apache
 
+Enable `mod_proxy`, `mod_proxy_http`, `mod_proxy_balancer`, `mod_headers`
+and `mod_lbmethod_byrequests`.
```
Comment (anoadragon453) [inline docs/reverse_proxy.md:102]: `mod_proxy_balancer` and `mod_lbmethod_byrequests` are likely only required if you require load balancing, which this example doesn't specify?

File: changelog.d/9517.doc [hunk 4]
```diff
@@ -0,0 +1 @@
+Complete instructions for Apache proxying
```
Comment (anoadragon453) [inline changelog.d/9517.doc:1]: You will need to add punctuation (just a period) to the end of this changelog entry.

**Review by anoadragon453 — CHANGES_REQUESTED** [review]
A minor change and a question, but otherwise lgtm. Learned a bit about Apache in the process too :slightly_smiling_face:

**Comment by anoadragon453** [comment]
Hey @crees, are you still up for helping drive this PR home?

**Comment by richvdh** [comment]
This seems to only need trivial changes, so I'll try and bring it up to date.

**Comment by richvdh** [comment]
actually it seems to have been superceded by #9696, so I'm going to close it.

**Pull request closed without merging**